pkg/smokescreen/smokescreen.go

```diff
@@ -240,20 +243,25 @@ func dialContext(ctx context.Context, network, addr string) (net.Conn, error) {
 	}
 
 	sctx.cfg.MetricsClient.Incr("cn.atpt.total", 1)
-	start := time.Now()
 
 	var conn net.Conn
 	var err error
 
+	start := time.Now()
 	if sctx.cfg.ProxyDialTimeout == nil {
 		conn, err = net.DialTimeout(network, d.resolvedAddr.String(), sctx.cfg.ConnectTimeout)
 	} else {
 		conn, err = sctx.cfg.ProxyDialTimeout(ctx, network, d.resolvedAddr.String(), sctx.cfg.ConnectTimeout)
 	}
+	connTime := time.Since(start)
+
+	fields := logrus.Fields{
+		"conn_establish_time": connTime.String(),
```

Comment:
I noticed this filed and the `dns_lookup_time` field aren't consistent in their unit of time they're logged in. Sometimes it's in `ms` and sometimes in microseconds (and I assume sometimes seconds if it takes long enough). Should we standardize the time before logging it for ease of sorting/searching?